hicommonwealth/commonwealth
# Uniswap trade as a standalone functionality

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/10217

## Description of Changes
- Renamed `FLAG_TOKENIZED_COMMUNITY` to `FLAG_LAUNCHPAD`
- Added pinned token trade functionality behind a new `FLAG_UNISWAP_TRADE` feature flag

## "How We Fixed It"
N/A

## Test Plan
- Disable `FLAG_LAUNCHPAD` and enabled `FLAG_UNISWAP_TRADE`
- Attempt to pin any token in an Ethereum community where you are admin ex: pin sushi `0x7d49a065d17d6d4a55dc13649901fdbb98b2afba`
- Verify you see this pinned token in the community sidebar
- Verify you are able to trade this token using the uniswap model.

## Deployment Plan
N/A

## Other Considerations
N/A
